src/lib/core/core.gni

```diff
@@ -91,6 +91,9 @@ declare_args() {
 
   # Whether the target architecture is big-endian (true) or little-endian (false).
   chip_target_is_big_endian = false
+
+  # dynamic cluster servers for controller
```

Comment:
This seems like a very strange thing to have in "core"....

Reply:
reverted it back, we just need set it in src/app/util/build.gn.. thanks